Source/ChilliSource/Rendering/Base/RenderFrameData.h

```diff
@@ -47,17 +47,7 @@ namespace ChilliSource
         
         RenderFrameData(RenderFrameData&&) = default;
         RenderFrameData& operator=(RenderFrameData&&) = default;
-        
-        /// Creates a new instance with the given viewport resolution and clear colour.
-        ///
-        /// @param frameAllocator
-        ///     The allocator which should be used for all frame allocations.
-        ///
-        RenderFrameData(IAllocator* frameAllocator) noexcept;
-        
-        /// @return The allocator which should be used for all frame allocations.
-        ///
-        IAllocator* GetFrameAllocator() const noexcept { return m_frameAllocator; }
+        RenderFrameData() noexcept;
```

Comment:
As with RenderFrame, this will need to either be a list of all allocated data within a single frame, or renamed to be more representative of what it now does.